version.go

```diff
@@ -11,10 +14,18 @@ const VersionMinor = 0
 // VersionPatch holds the release patch number
 const VersionPatch = 0
 
-// Version holds the combination of major minor and patch as a string
-// of format Major.Minor.Patch
-var Version = fmt.Sprintf("%d.%d.%d", VersionMajor, VersionMinor, VersionPatch)
```

Comment:
@moypray If you set a git tag, it will show up. This works great and I've used it on several projects with success.